apps/nextjs-website/src/components/atoms/UserInfo/UserInfo.tsx

```diff
@@ -1,15 +1,52 @@
 'use client';
-import { useAuthenticator } from '@aws-amplify/ui-react';
+import { translations } from '@/_contents/translations';
+import { PersonOutline } from '@mui/icons-material';
 import { Button, Stack, Hidden, Typography } from '@mui/material';
-import { FC } from 'react';
+import { Auth, Hub } from 'aws-amplify';
+import { FC, useCallback, useEffect, useState } from 'react';
 
 const UserInfo: FC = () => {
-  const { signOut, user, authStatus } = useAuthenticator((context) => [
-    context.user,
-    context.authStatus,
-  ]);
+  const { shared } = translations;
+  const [user, setUser] = useState<any>(null);
```

Comment:
You can use more precise types here

Reply:
Unfortunately `aws-amplify` [doesn't expose the correct `user`'s type](https://github.com/aws-amplify/amplify-js/issues/4927), should we create a custom type?

Reply:
😣. Yes let's validate the structure of the user (e.g. using `io-ts`). Choose yourself between create a ticket or do it now. If you choose the latest add also a TODOs 🙏

Reply:
I added the user type, but I'd leave the validation for another ticket